docs/proposals/cli/README.md

```diff
@@ -0,0 +1,143 @@
+# orkactl
+
+## Requirements
+
+For the v0 of orka, only some services are available, which you can see below:
```

Comment:
v0, v1, etc are internal milestones that this document should not bother about.

Here we want to get the component overall requirements, scope and architecture. What will be achieved with v0 is a project management and scheduling issue.